Description: Change path for shared files
 We use /usr/share/games/.
Author: Tobias Hansen <thansen@debian.org>

--- a/nall/string/platform.hpp
+++ b/nall/string/platform.hpp
@@ -80,7 +80,7 @@
   #elif defined(PLATFORM_OSX)
   result = "/Library/Application Support/";
   #else
-  result = "/usr/share/";
+  result = "/usr/share/games/";
   #endif
   if(result.empty()) result = ".";
   if(result.endswith("/") == false) result.append("/");
